docs/user_group_management.md

```diff
@@ -0,0 +1,61 @@
+## Management in basic authentication mode Guide
+
+
+In the old version of OpenPAI, when deployed in basic authentication mode, pai didn't provide explicit group information for the admins, thus raising confusion when managing the platform. In latest version(>=0.16.0), pai will provide direct group management.
+
+This document will introduce how to manage user, group and teamwise storage in OpenPAI basic authentication mode in the latest version(>=0.16.0).
+
+### User Management
+#### Create a new user
+
+To add a new user to the cluster, cluster admin needs to do:
+
+- create user using [create user api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/createUser)
+- users' username and password must be given when creating a user. 
+
+#### Update existing user
+
+
+In basic authentication mode, admin can update the existing users' information directly.
+
+
+- User name, password and email can be changed by using [user apis](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#tag/user).
+- User's grouplist can be updated by using [user group apis](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/updateUserGrouplist). Only admins can update users' group information.
+
+
+#### Delete a user
+- delete user using [delete user api](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/deleteUser)
+
+
+### Group management
+
+
+In basic authentication mode, a group named 'default' and a group named 'admingroup' will be created when deploy OpenPAI. Every user needs to be in a group to have access to VCs and teamwise storages.
+It is possible to create a user without assigning any group to it. This user will be assigned to group default. Groups that are admin have access to all the VCs and teamwise storages.
```

Comment:
group default -> group `default`.
Groups that are admin -> And `admingroup` has the permission to access all the VCs and ...